motr/cob.c

```diff
@@ -268,6 +268,7 @@ static void cob_req_release(struct cob_req *cr) {
 		op->op_priv = NULL;
 	m0_free(cr->cr_ios_fop);
 	m0_free(cr->cr_ios_replied);
+	m0_free(cr->cr_cob_attr);
```

Comment:
I see that we are also releasing memory for cr_name at other similar places. Could you check if that is needed here too?
	m0_free(cr->cr_name.b_addr);
	M0_SET0(&cr->cr_name);

Reply:
Hi @mehjoshi ,  There is no leak with cr_name.   No need to free it in cob_req_release(). 

Only places where we are allocating memory to cr_name are...
  1) m0__obj_attr_get_sync()
  2) m0__obj_layout_send()

And both these functions taking care of freeing that memory.

Reply:
ok.